src/test/run-fail/async-fn-resume-after-completion.rs

```diff
@@ -0,0 +1,43 @@
+// issue 65419 Attempting to run an async fn after completion mentions generators when it should
```

Comment:
Please move all of the new tests into `src/test/ui/async-await/issues/` and use `// run-fail` to get the desired behavior. Also please mention the issue number in the file name as well, e.g. `issue-65419-async-fn-resume-after-completion.rs`.

Reply:
Much nicer that way, thanks I'll do that!

Reply:
@Centril 
> Please move all of the new tests into src/test/ui/async-await/issues/

Unfortunately this broke `--pass check`: https://github.com/rust-lang/rust/issues/66929